generators/client/templates/src/main/webapp/app/_app.config.js

```diff
@@ -1,49 +1,51 @@
-'use strict';
+(function() {
+  'use strict';
```

Comment:
I would prefer it inside the closure which is recommended by jslint, jshint etc as well